database/src/lib.rs

```diff
@@ -664,7 +703,7 @@ impl Index {
         self.pstat_series
             .map
             .keys()
-            .map(|(_, _, _, _, metric)| metric)
+            .map(|(_, _, _, _, metric, _)| metric)
```

Comment:
```suggestion
            .map(|(_, _, _, _, _, metric)| metric)
```
Otherwise it would actually select the target.